### ISSUE TITLE
Show sub-collections under parent collection

### ISSUE THREAD

**Issue by SoCuul** [linkwarden/linkwarden#634]
**Is your feature request related to a problem? Please describe.**
When viewing a collection's page, it only lists the links stored within it, not showing any sub-collections. This provides a very unintuitive experience while using sub-collections (especially on mobile).

Clicking on a collection currently with only sub-collections (no links) looks like it's empty, even though it already has several sub-collections present within it.

**Describe the solution you'd like**
It would be extremely nice if sub-collections were displayed on the same page alongside links, to act as an alternative to navigating via the sidebar.

On mobile, this would provide a substantially better way to navigate through the collection tree. It would also be **extremely** nice if this was provided as an optional toggle, to give users the option to use either display style.

**Describe alternatives you've considered**
There is sub-collection navigation provided by the sidebar, but this is inconvenient (especially if it's hidden on mobile).

**Additional context**
I have provided a mockup of what this feature could possibly resemble, though this was thrown together quickly and is not meant to represent a final UI layout.

![](https://github.com/linkwarden/linkwarden/assets/63339559/893cc96f-a58c-440a-b386-042561eeafc2)

**Comment by SoCuul** [comment]
This would also resolve the problems present in #610

**Comment by CaymanIslands** [comment]
Yes, this feature would be a bog improvement to the usability, and it would be an excellent opportunity to:

(1) vertically align all of the subcategory "folder" icons in the tree by drawing the child indicators (triangles) as part of the folder icon image (superimposed) for an always-single folder icon.

(2) auto-sort the tree nodes alphabetically.

Those improvements would make navigation much easier and faster on both desktop and mobile.

**Comment by SoCuul** [comment]
@daniel31x13 👀

**Comment by sofmeright** [comment]
I am really hopeful that if this does get implemented it can also allow the parent collection to inherit all the child collections that have also been marked public or even better to have the ability to share a whole parent tree including everything beneath,

**Comment by daniel31x13** [comment]
Added in v2.9.0 (thanks to @KittyKatt), enjoy!
